src/bridge/cockpitnetworksamples.c

```diff
@@ -92,14 +89,8 @@ cockpit_network_samples (CockpitSamples *samples)
 
       cockpit_samples_sample (samples, "network.interface.rx", iface_name, bytes_rx);
       cockpit_samples_sample (samples, "network.interface.tx", iface_name, bytes_tx);
-
-      total_rx += bytes_rx;
-      total_tx += bytes_tx;
     }
 
-  cockpit_samples_sample (samples, "network.all.rx", NULL, total_rx);
-  cockpit_samples_sample (samples, "network.all.tx", NULL, total_tx);
```

Comment:
Isn't this (borderline) breaking API? Are we sure we're OK with that? @petervo what do you think?